Explorer/Assets/DCL/CharacterMotion/Animation/ApplyJumpState.cs

```diff
@@ -0,0 +1,43 @@
+﻿using DCL.AvatarRendering.AvatarShape;
+using DCL.CharacterMotion.Components;
+using DCL.CharacterMotion.Settings;
+using System.Runtime.CompilerServices;
+
+namespace DCL.CharacterMotion.Animation
+{
+    public static class ApplyJumpState
+    {
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static void Execute(ref CharacterAnimationComponent animationComponent,
+            in ICharacterControllerSettings settings,
+            in CharacterRigidTransform rigidTransform,
+            in AvatarBase avatarBase,
```

Comment:
I propose to create an interface for `AvatarBase.avatarAnimator` to enable Testability. And in both this method and `ApplyMovementBlend` you don't really need to couple with `MonoBeahviours` (especially considering `AvatarBase` is a draft